src/Datadog.Trace/Sampling/RateLimiter.cs

```diff
@@ -59,7 +59,7 @@ public bool Allowed(Span span)
 
                 if (count >= _maxTracesPerInterval)
                 {
-                    Log.Debug("Dropping trace id {0} with count of {1} for last {2}ms.", span.TraceId, count, _intervalMilliseconds);
+                    Log.Warning("Dropping trace id {TraceId} with count of {Count} for last {Interval}ms.", span.TraceId, count, _intervalMilliseconds);
```

Comment:
This may be very noisy if it's a warning level.

Reply:
\*Shakes fist at RFC\*

Reply:
With the rate limiting API, (in a separate PR atm) this shouldn't be a problem